modules/ballerina-core/src/main/java/org/ballerinalang/model/builder/BLangModelBuilder.java

```diff
@@ -1301,11 +1301,6 @@ public void createReturnStmt(NodeLocation location, WhiteSpaceDescriptor whiteSp
 
     public void createReplyStmt(NodeLocation location, WhiteSpaceDescriptor whiteSpaceDescriptor) {
         Expression argExpr = exprStack.pop();
-        if (!(argExpr instanceof SimpleVarRefExpr)) {
-            String errMsg = BLangExceptionHelper.constructSemanticError(location,
-                    SemanticErrors.REF_TYPE_MESSAGE_ALLOWED);
```

Comment:
SemanticErrors.REF_TYPE_MESSAGE_ALLOWED is no longer using. Shall we remove that as well?